minecraft_servers/cubehub/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "server_name": "cubehub",
+  "nice_name": "CubeHub",
+  "direct_ip": "CubeHub.dk",
+  "server_wildcards": [
+    "cubehub.dk",
+    "%.cubehub.dk"
+  ],
+  "supported_languages": [
+    "dk"
+  ],
+  "social": {
+    "web": "https://cubehub.dk",
+    "web_shop": "https://butik.cubehub.dk",
+    "web_support": "https://forum.cubehub.dk",
```

Comment:
Website is a mail login; support cannot be forum; add FAQ or contact form, remove if non-existent